$NetBSD: patch-Makefile,v 1.2 2015/07/16 08:57:32 leot1990 Exp $

Convert the fmt(1) command to a tr(1) one (the fmt(1) old syntax command is not
supported on all Unix systems).

--- Makefile.orig	2015-07-16 08:17:54.000000000 +0000
+++ Makefile
@@ -276,8 +276,8 @@ override CFLAGS += -DFIO_VERSION='"$(FIO
 	@$(CC) -MM $(CFLAGS) $(CPPFLAGS) $*.c > $*.d
 	@mv -f $*.d $*.d.tmp
 	@sed -e 's|.*:|$*.o:|' < $*.d.tmp > $*.d
-	@sed -e 's/.*://' -e 's/\\$$//' < $*.d.tmp | fmt -1 | \
-		sed -e 's/^ *//' -e 's/$$/:/' >> $*.d
+	@sed -e 's/.*://' -e 's/\\$$//' < $*.d.tmp | tr -cs "[:graph:]" "\n" | \
+		sed -e 's/^ *//' -e '/^$$/ d' -e 's/$$/:/' >> $*.d
 	@rm -f $*.d.tmp
 
 ifdef CONFIG_ARITHMETIC
